### PR TITLE
Improved LoginScreen UI

### PR DESCRIPTION
**Pull request by saketkumar95**
This PR Improves UI of Login screen

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the checks with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.
screen shot 2018-02-28 at 7 38 08 pm
@sagar15795 Please review this. :)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -97,6 +97,7 @@ dependencies {
         exclude module: 'xpp3'
     }
 
+    implementation 'com.android.support:customtabs:25.0.0'
```
Comment (sagar15795) [inline app/build.gradle]: use latest support library version and manage it using `$rootProject.supportLibraryVersion`

File: app/src/main/res/layout/fragment_login.xml [hunk 2]
```diff
@@ -42,10 +44,11 @@
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:layout_below="@+id/logo"
+            android:typeface="serif"
             android:layout_centerHorizontal="true"
-            android:layout_marginTop="42dp"
+            android:layout_marginTop="12dp"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_login.xml]: move all margin, padding and text size to `dimens.xml`.

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -97,7 +97,8 @@ limitations under the License.
     <string name="loading">Loading details</string>
     <string name="loadworkflows">Loading Workflows</string>
     <string name="fetchrun">Fetching run components</string>
-    <string name="keeplogin">Remain Logged-in</string>
+    <string name="keeplogin">Remember me</string>
```
Comment (sagar15795) [inline app/src/main/res/values/strings.xml]: `Remain Logged-in` is more intuitive than `Remember me`. 
Please Revert this.

File: app/src/main/res/layout/fragment_login.xml [hunk 4]
```diff
@@ -110,17 +115,28 @@
...
-                android:textSize="18sp"/>
+                android:text="@string/bLogin" />
 
+            <android.support.v7.widget.AppCompatButton
+                style="@style/Widget.AppCompat.Button.Borderless"
+                android:id="@+id/bRegister"
+                android:layout_height="wrap_content"
+                android:layout_width="match_parent"
+                android:text="@string/create_an_account"
+                android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_login.xml]: set text size smaller than login's text size.

File: app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java [hunk 5]
```diff
@@ -162,6 +167,20 @@ public void login(View v) {
         }
     }
 
+    @OnClick(R.id.bRegister)
+    public void register(View v) {
+        if (Build.VERSION.SDK_INT < 15) {
+            Intent i = new Intent(Intent.ACTION_VIEW);
```
Comment (sagar15795) [inline app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java]: use `intent` in place of `i`

File: app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java [hunk 6]
```diff
@@ -162,6 +167,20 @@ public void login(View v) {
...
+            i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            i.setData(Uri.parse(myExperimentURL));
+            startActivity(i);
+        } else {
+            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
+            CustomTabsIntent i = builder.build();
```
Comment (sagar15795) [inline app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java]: use `customTabsIntent` in place of `i`

File: app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java [hunk 7]
```diff
@@ -162,6 +167,20 @@ public void login(View v) {
...
 
+    @OnClick(R.id.bRegister)
+    public void register(View v) {
+        if (Build.VERSION.SDK_INT < 15) {
+            Intent i = new Intent(Intent.ACTION_VIEW);
+            i.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
+            i.setData(Uri.parse(myExperimentURL));
+            startActivity(i);
+        } else {
+            CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
```
Comment (sagar15795) [inline app/src/main/java/org/apache/taverna/mobile/ui/login/LoginFragment.java]: use `customTabsIntentBuilder` in place of `builder`

**Comment by saketkumar95** [comment]
@sagar15795 Done! 👍

File: app/src/main/res/layout/fragment_login.xml [hunk 8]
```diff
@@ -42,42 +44,43 @@
...
             android:orientation="vertical"
-            android:padding="20dp">
+            android:padding="@dimen/layout_login">
 
             <TextView
                 android:id="@+id/tvLoginInfo"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:layout_gravity="center"
                 android:background="#E1E1E1"
```
Comment (sagar15795) [inline app/src/main/res/layout/fragment_login.xml]: Move it to `color.xml`

**Comment by sagar15795** [comment]
@saketkumar95 
Please make the requested change.

**Comment by saketkumar95** [comment]
@sagar15795 Done! 👍

**Comment by sagar15795** [comment]
@saketkumar95 Thanks for the UI improvement.

**Pull request merged**